modules/ocf_postgres/manifests/init.pp

```diff
@@ -0,0 +1,44 @@
+class ocf_postgres {
+
+  include ocf::ssl::default
+
+  class { 'postgresql::server':
+    postgres_password => hiera('postgres::rootpw'),
+    # https://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html
+    ipv4acls          => ['hostssl sameuser all 0.0.0.0/0 md5'],
+    ipv6acls          => ['hostssl sameuser all ::/0 md5'],
+  }
+
+  file {
+    # copies proper .pgpass file for ocfbackups to authenticate on backup
+    '/opt/share/.pgpass':
+      source    => 'puppet:///private/backups/.pgpass',
```

Comment:
Note that the correct path for the source should actually be `puppet:///private/backups/pgpass` (without the dot).